# HG changeset patch
# User Rik <rik@octave.org>
# Date 1346804731 25200
#      Tue Sep 04 17:25:31 2012 -0700
# Node ID 2981cdbb955f6c54eeecc082b0f8ab6f386544a8
# Parent  637e3b7d6a56024d33815f466881d914970e8899
acinclude.m4: Correct message for OCTAVE_CHECK_FUNC_FIRDFIRST_MODERN.

* acinclude.m4: Correct message for OCTAVE_CHECK_FUNC_FIRDFIRST_MODERN.

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -144,17 +144,17 @@ AC_DEFUN([OCTAVE_CHECK_FUNC_CMATH], [
 ])
 dnl
 dnl Check whether Qscintilla FindFirst function is old (16 inputs) or
 dnl new (17 inputs).
 dnl FIXME: This test uses a version number.  It potentially could
 dnl        be re-written to actually call the function, but is it worth it?
 dnl
 AC_DEFUN([OCTAVE_CHECK_FUNC_FINDFIRST_MODERN], [
-  AC_CACHE_CHECK([whether Qscintilla FindFirst uses 16 or 17 input arguments],
+  AC_CACHE_CHECK([whether Qscintilla FindFirst uses 17 input arguments],
     [octave_cv_func_findfirst_modern],
     [AC_LANG_PUSH(C++)
     save_CPPFLAGS="$CPPFLAGS"
     CPPFLAGS="-I$QT_INCDIR -I$QT_INCDIR/Qt $CPPFLAGS"
     AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
         #include <Qsci/qsciglobal.h>
         ]], [[
         #if QSCINTILLA_VERSION < 0x020600
